src/loaders/storage/loadEnvironment.ts

```diff
@@ -0,0 +1,15 @@
+import { LoadingManager, DataTexture } from "three"
+
+import { loadRGBE } from "../operators/loadRGBE"
+// import { loadEXR } from "../operators/loadEXR"
+import { storageMemoize } from "./storageMemoize"
+
+export const loadEnvironment = (loadingManager?: LoadingManager) =>
+  storageMemoize(async () => {
+    const { default: hdr } = await import(
+      // @ts-ignore
+      /* webpackChunkName: "Environment" */ "../../assets/models/textures/Environment/Environment.hdr"
```

Comment:
Has this made it into [SharedAssets](https://github.com/SaltieRL/SharedAssets) yet?

Reply:
No, I haven't finished final render yet, current environment texture is a testrender I made.

Reply:
I think it is okay to put the test render in.  It is honestly high quality enough for temporary use.

@Abbondanzo is there a way to flag the render under beta?

Reply:
>is there a way to flag the render under beta?

Not entirely, but we can flag the version number in `package.json` as `0.7.0b` or something of that nature. Publishing only happens once we cut releases in this repo